scripts/ingest_dfc/utils.py

```diff
@@ -153,6 +153,17 @@ def _key(key: str) -> str:
         return key.lower().replace(" ", "_").replace("?", "").replace("y/", "")
 
 
+    def get_first_url(self) -> str:
+        """Gets the first URL from the 'documents' attribute.
+        
+        TODO: This could/should be written with more validation.
+        """
+        first_doc = self.documents.split(';')[0]
+        first_url = first_doc.split('|')[0]
+        return first_url
```

Comment:
Is it worth warning/erroring if we find more than one document?

Reply:
Yeah - good point - should I raise ?